components/idp-mgt/org.wso2.carbon.idp.mgt.ui/src/main/resources/web/idpmgt/idp-mgt-edit.jsp

```diff
@@ -3667,6 +3694,18 @@
                                         </div>
                                     </td>
                                 </tr>
+                                <tr>
+                                    <td class="leftCol-med labelField"><fmt:message key='name.id.format'/>:<span
+                                            class="required">*</span></td>
+                                    <td>
+                                        <input id="NameIDType" name="NameIDType" type="text" size="70"
```

Comment:
I'm assuming this is the place where we enter NameIDType in UI. Any idea whether the number of NameIDTypes  is fixed or not ? If it's fixed we can use a combo box.